CONTRIBUTING.md

```diff
@@ -102,6 +102,15 @@ Closes #318
 Update **`Start working on Issues`** section with clearer instructions on getting ready to work on an issue.
 ```
 
+### GitHub Action
+
```

Comment:
Please update the TOC for this section.

Reply:
okay. just pushed another [commit](https://github.com/kawalcovid19/wargabantuwarga.com/pull/742/commits/08d51d96f01531e5c6ce7b59649f400fa8259147)